docs/specifications/tests/Delegation-TP/delegation02.md

```diff
@@ -1,48 +1,76 @@
-## DELEGATION02: Name servers must have distinct IP addresses
+# DELEGATION02: Name servers must have distinct IP addresses
 
-### Test case identifier
+## Test case identifier
 
 **DELEGATION02:** Name servers must have distinct IP addresses
 
-### Objective
+## Objective
 
 If the domain have several different name server names used, they can all
 be using the same IP address. This may be due to a configuration error, or
 a workaround to a certain policy restriction. This test case checks that
 the name servers used does not resolve to reuse the same IP addresses.
 
-Section 4.1 of [RFC 1034](https://tools.ietf.org/html/rfc1034) says at least
+Section 4.1 of [RFC 1034] says at least
 to name servers must be used for a delegation.
 
-### Inputs
+## Inputs
 
 The domain name to be tested.
 
-### Ordered description of steps to be taken to execute the test case
+## Ordered description of steps to be taken to execute the test case
 
-1. Obtain the IP addresss of the name servers from the parent using [Method
-4](../Methods.md#method-4-obtain-glue-address-records-from-parent) and the child using
-   [Method 5](../Methods.md#method-5-obtain-the-name-server-address-records-from-child).
-2. If any of the IP addresses resolved in step 1 is not unique, then this
-   test case fails.
+1. Obtain the complete set of name servers (distinct NS records) from 
+   parent using [Method2].
+2. Obtain the IP addresses for the set of the name servers from parent 
+   using [Method4]. (This should include all IP address
+   for both in-bailiwick and out-of-bailiwick name servers.)
+3. If the same IP address is found for two or more name servers then 
+   emit *PARENT_NS_SAME_IP_ADDRESS* and the IP address.
+4. Repeat step 3 for all IP addresses obtained in step 2.
+5. Obtain the complete set of name servers from the child using 
+   [Method3].
```

Comment:
What are these names used for? Either some subsequent step should refer to this result, or this step should be removed.

Reply:
You need to have both name and address to see that  more than one name uses the same address. If you just get a set of IP addresses then you cannot see it.